UT4MasterServer.Services/Scoped/RatingsService.cs

```diff
@@ -206,6 +207,7 @@ public async Task UpdateDeathmatchRatingsAsync(RatingMatch ratingMatch)
 			.Select(s => EpicID.FromString(s.AccountID))
 			.ToArray();
 		int playersCount = playersAccountIds.Length;
+		if (playersCount <= 1) return;
```

Comment:
maybe change to `< 2` to make it slightly more clear when glancing over the code
```suggestion
		if (playersCount < 2) return;
```